proto/meta.proto

```diff
@@ -94,6 +94,14 @@ message TableFragments {
     // so we pre-generate and store it here, this member will only be initialized when creating the Fragment
     // and modified when creating the mv-on-mv
     repeated uint32 upstream_fragment_ids = 7;
+
+    // Total vnode count of the fragment (then all internal tables).
+    // Duplicated from the length of the vnode bitmap in any actor of the fragment.
+    //
+    // Can be unset if the fragment is created in older versions where variable vnode count is not
+    // supported, in which case a default value of 256 should be used.
+    // Use `VnodeCountCompat::vnode_count` to access it.
+    optional uint32 maybe_vnode_count = 8;
```

Comment:
The reason why there's a `maybe_` prefix is that, a getter method with the same name as the field will be generated for `prost` structs. Directly naming it `vnode_count` will lead to the method `vnode_count()` returning `0` when the field is unset, which can be misleading sometimes.

In the current approach, when the developer write `vnode_count()`, it will be forwarded to `VnodeCountCompat::vnode_count`, which correctly handles the backwards compatibility.